### PR TITLE
COMMERCE-12613 Show correct UI for when pricing is Price on Application

### PR DESCRIPTION
**Pull request by brianikim**
@andrea-ale-sbarra ,

https://liferay.atlassian.net/browse/COMMERCE-12613

### CODE REVIEW

**Review by andrea-ale-sbarra — COMMENTED** [review]
@smottal can you please have a look at this? I'm not sure I have the right behaviour of this in my mind.
It looks good to me, but I may have a wrong idea.
Thank you

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItemUnitOfMeasure.js [hunk 1]
```diff
@@ -429,8 +430,16 @@ const PriceRows = ({price}) => {
 						</span>
 					</PriceRow>
 				</div>
-			) : (
-				<></>
+			)}
+
+			{price && priceOnApplication && (
+				<div className="mini-cart-prices mt-4">
+					<PriceRow>
```
Comment (smottal) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItemUnitOfMeasure.js:437]: Is it correct to don't have the "label" on the left if it's price on application?

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js [hunk 2]
```diff
@@ -194,53 +195,69 @@ function EditItem() {
...
-								{price.finalPriceFormatted}
-							</span>
-						</PriceRow>
-					</div>
+						</div>
+					)}
+
+					{priceOnApplication && (
+						<div className="mini-cart-prices p-4">
+							<PriceRow>
```
Comment (smottal) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js:252]: Is it correct to don't have the "label" on the left if it's price on application?

**Comment by smottal** [comment]
@andrea-ale-sbarra @brianikim just a comment because I don't know how the value should be visualized in case of price on application. The rest of the code seems to be fine

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItemUnitOfMeasure.js (hunk 1 above)
Comment (brianikim) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItemUnitOfMeasure.js:437]: I thought about this @smottal , but I wasn't sure what the correct "label" would look like. I simply followed our current minicart behavior where the item does not show the price, but rather have Price On Application instead. I created a thread to check with Bernard and David

**Comment by brianikim** [comment]
@andrea-ale-sbarra , @smottal , reassigned for review. Bernard agreed current change is good.

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
Resent at https://github.com/liferay-commerce/liferay-portal/pull/4424

**Comment by Brunno0liveira** [comment]
ci:close

**Pull request closed without merging**